props/blaster_BC_buttons.h

```diff
@@ -538,11 +538,19 @@ class BlasterBCButtons : public Blaster {
     Off();
   }
 
-  // Pull in parent's SetPreset, but set the correct alt sounds.
-  void SetPreset(int preset_num, bool announce) override {
-    Blaster::SetPreset(preset_num, announce);
-    SaberBase::DoEffect(EFFECT_ALT_SOUND, 0.0, blaster_mode);
+void ResetCurrentAlternative() override {
```

Comment:
why not just current_alternative = (int)BLASTER_DEFAULT_MODE; ?

Reply:
No reason other than I'm not that slick.
Thanks. Done.